examples/detection_callback.py

```diff
@@ -0,0 +1,34 @@
+"""
+Detection callback w/ scanner
+--------------
+
+Example showing what is returned using the callback upon detection functionality
+
+Updated on 2020-10-11 by bernstern <bernie@allthenticate.net>
+
+"""
+
+import asyncio
+from bleak import BleakScanner
+from bleak.backends.scanner import AdvertisementData
+import logging
+
+logging.basicConfig()
+
+
+def simple_callback(callback_data: AdvertisementData):
+    print(callback_data)
```

Comment:
it would be nice to print all data here (e.g. the suggested `__repr__` method above)